packages/connect-core/src/entities/Role.ts

```diff
@@ -31,7 +33,7 @@ export default class Role extends CoreEntity {
 
     const { roles } = metadata[0] as AragonArtifact
 
-    const role = roles.find(role => role.bytes === data.hash)
+    const role = roles?.find(role => role.bytes === data.hash)
```

Comment:
We had an error here when the `roles` were undefined.

Reply:
Ah yes and we are using `as` before so TS couldn’t catch it :s